### PR TITLE
minor fixes and updates

### PR DESCRIPTION
**Pull request by zroubalik**
Signed-off-by: Zbynek Roubalik <zroubali@redhat.com>

1. The filter on the `responder` trigger needs to be set this way. Filters on a trigger work as logical AND, so the CloudEvent would have to have both types set.
2. Removed Azure url, from the Face API call, it is not needed and caused problems with not being resolvable
3. Added an image with a diagram of the deployed app
4. Minor nits in the functions code

### CODE REVIEW

File: deploy/trigger-responder.yaml [hunk 1]
```diff
@@ -2,18 +2,30 @@
 apiVersion: eventing.knative.dev/v1
 kind: Trigger
 metadata:
-  name: trigger-responder
+  name: trigger-responder-image
 spec:
   broker: default
   filter:
-    and:
```
Comment (matejvasek) [inline deploy/trigger-responder.yaml:9]: shouldn't this have been `or`?

Comment (zroubalik) [reply deploy/trigger-responder.yaml:9]: The previous version with AND don't work anyway 😄

File: processor/src/main/java/functions/Function.java [hunk 2]
```diff
@@ -50,9 +50,8 @@ private void getData(Input input, CloudEvent cloudEvent, UniEmitter<? super Outp
         if (retries <= 0) {
             emitter.fail(new RuntimeException("Too many fails to call face api."));
         }
-        FaceAPI faceAPI = FaceAPIManager.authenticate("https://boson.cognitiveservices.azure.com/face/v1.0", apiKey);
+        FaceAPI faceAPI = FaceAPIManager.authenticate(AzureRegions.EASTUS, apiKey);
```
Comment (lance) [inline processor/src/main/java/functions/Function.java:53]: Is the `boson` subdomain unnecessary (I assume it's not part of `AzureRegions.EASTUS`)?

**Review by lance — APPROVED** [review]
LGTM

File: processor/src/main/java/functions/Function.java (hunk 2 above)
Comment (zroubalik) [reply processor/src/main/java/functions/Function.java:53]: Yeah, it is not needed in order to make API calls

**Pull request merged**